src/System.Private.ServiceModel/src/System/ServiceModel/Channels/ProducerConsumerStream.cs

```diff
@@ -190,9 +190,9 @@ private static void CancelAndDispose(object state)
             }
         }
 
-#pragma warning disable 660,661  // Hashcode not needed, Equals overriden for fast path comparison of EmptyContainer
+#pragma warning disable 659,660,661  // Hashcode not needed, Equals overriden for fast path comparison of EmptyContainer
         private struct WriteBufferWrapper : IEquatable<WriteBufferWrapper>
-#pragma warning restore 660,661
+#pragma warning restore 659,660,661
```

Comment:
Question only: 

According to MSDN: 
> CS0659:  'class' overrides Object.Equals(object o) but does not override Object.GetHashCode()
> CS0660: 'class' defines operator == or operator != but does not override Object.Equals(object o)
> CS0661: 'class' defines operator == or operator != but does not override Object.GetHashCode()

Seems to me that CS0660 is no longer needed here? 
Also, what's the effect of the `pragma warning disable 659` on Line 228? Is it needed in both the class definition and the method?